tests/make-targets-updated.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/node
+
+const path = require('path');
+const colors = require('colors');
+
+
+try {
+  require('child_process').execSync(`make -B -C ${path.join(__dirname, '..')}`);
```

Comment:
Why do you have to `make` unconditionally? The normal implementation should suffice. In either case, add a comment about -B and -C or use long forms if possible